doc/specs/#888 - Com Api.md

```diff
@@ -0,0 +1,889 @@
+        AllLocalPackages,
+        /// Search both local and remote sources.
+        AllPackages,
+    };
+
+    /// IMPLEMENTATION NOTE: MatchType from AppInstallerRepositorySearch.h
+    [contract(Microsoft.Management.Deployment.WindowsPackageManagerContract, 1)]
+    enum MatchType
+    {
+        Exact,
+        CaseInsensitive,
+        StartsWith,
+        Fuzzy,
+        Substring,
+        FuzzySubstring,
+        Wildcard,
+    };
+    /// IMPLEMENTATION NOTE: PackageMatchField from AppInstallerRepositorySearch.h
+    /// The field to match on.
+    /// The values must be declared in order of preference in search results.
+    [contract(Microsoft.Management.Deployment.WindowsPackageManagerContract, 1)]
+    enum PackageMatchField
+    {
+        AppCatalogDefined,
+        Id,
+        Name,
+        Moniker,
+        Command,
+        Tag,
+        /// DESIGN NOTE: The following PackageMatchField from AppInstallerRepositorySearch.h are not implemented in V1.
+        /// PackageFamilyName,
+        /// ProductCode,
+        /// NormalizedNameAndPublisher,
+    };
+    /// IMPLEMENTATION NOTE: PackageMatchFilter from AppInstallerRepositorySearch.h
+    [contract(Microsoft.Management.Deployment.WindowsPackageManagerContract, 1)]
+    runtimeclass PackageMatchFilter
+    {
+        PackageMatchFilter();
+        /// Whether the CatalogPackage must match the filter in order to be included in the matches.
+        Boolean IsAdditive;
+        /// The type of string comparison for matching
+        MatchType Type;
+        /// The field to search
+        PackageMatchField Field;
+        /// The value to match
+        String Value;
+        /// DESIGN NOTE: "Additional" from RequestMatch AppInstallerRepositorySearch.h is not implemented here.
+    };
+
+    /// IMPLEMENTATION NOTE: ResultMatch from AppInstallerRepositorySearch.h
+    /// A single result from the search.
+    [contract(Microsoft.Management.Deployment.WindowsPackageManagerContract, 1)]
+    runtimeclass ResultMatch
+    {
+        /// The package found by the search request.
+        CatalogPackage CatalogPackage { get; };
+
+        /// The highest order field on which the package matched the search.
+        PackageMatchFilter MatchCriteria { get; };
+    }
+    /// IMPLEMENTATION NOTE: SearchResult from AppInstallerRepositorySearch.h
+    /// Search result data returned from FindPackages
+        [contract(Microsoft.Management.Deployment.WindowsPackageManagerContract, 1)]
+    runtimeclass FindPackagesResult
+    {
+        /// The full set of results from the search.
+        Windows.Foundation.Collections.IVectorView<ResultMatch> Matches { get; };
+
+        /// If true, the results were truncated by the given SearchRequest::MaximumResults.
+        /// USAGE NOTE: Windows Package Manager does not support result pagination, there is no way to continue 
+        /// getting more results.
+        Boolean IsTruncated{ get; };

```

Comment:
IsPartial? IsIncomplete? WasLimitExceeded?

Reply:
RECOMMEND: `WasLimitExceeded` - the catalog decided to only give you this many.